web/pgadmin/browser/server_groups/servers/pg_timetable/tests/utils.py

```
@@ -0,0 +1,320 @@
+            ) has_connstr
+        """
+        pg_cursor.execute(SQL)
+        result = pg_cursor.fetchone()
+        if result is None:
+            connection.close()
+            message = "Make sure pgTimetable is installed properly."
+            return False, message
+
+        connection.close()
+        return True, None
+    except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)